ckanext/datastore/backend/postgres.py

```diff
@@ -1672,6 +1678,33 @@ def _read_connection_has_correct_privileges(self):
             write_connection.close()
         return True
 
+    def _write_url_search_connection_has_correct_privileges(self):
```

Comment:
How often does this function execute? Just at boot? Every time a function uses the read engine?

Reply:
Just at boot. If the pod or apache service restarts then this check is run.